gst/nnstreamer/tensor_query/tensor_query_serversrc.h

```diff
@@ -40,7 +38,7 @@ typedef struct _GstTensorQueryServerSrcClass GstTensorQueryServerSrcClass;
  */
 struct _GstTensorQueryServerSrc
 {
-  GstPushSrc element; /* parent object */
+  GstPushSrc element;           /* parent object */
```

Comment:
```suggestion
  GstPushSrc element; /* parent object */
```
Any reason?

Reply:
Reverted. It's automatically changed by gst-indent tool.